changelogs/fragments/251-terminology-changes.yaml

```diff
@@ -0,0 +1,7 @@
+new_modules:
+- New module ``command_parser`` (renamed from ``text_parser``)
+- New module ``textfsm_parser`` (renamed from ``textfsm``)
+
+deprecated_features:
+- Module ``text_parser`` deprecated; legacy use supported; will be removed in 2.6.0.
+- Module ``textfsm`` deprecated; legacy use supported; will be removed in 2.6.0.
```

Comment:
@acozine @gundalow We also will need `bugfixes` section in here which should be in CHANGELOG.rst also after building.
`bugfixes` should have link to the PRs fixed for the issues opened.